### DISCUSSION TITLE
qfield DCIM

### DISCUSSION THREAD

**Discussion by Robertotragsa** [opengisch/QField#3917] [Q&A]
Hi! I am working on a coastal project and all I need is that the photos taken on the field go to a predetermined folder related to a QGIS 
"beach" field. How should I proceed?

**Comment by lucienicolier** [comment]
Hi,

You need to add a folder with your pictures in the same folder as your project when you sync your project on the cloud.

[pictures](https://docs.qfield.org/how-to/pictures/?h=pic#use-pictures)
[pictures path](https://docs.qfield.org/how-to/pictures/?h=pictu#configurable-picture-path)

Best
